features/step_definitions/your_results_frequency_selector_steps.rb

```diff
@@ -0,0 +1,31 @@
+When(/^I enter my age as 25$/) do
+  your_details_page.age.set(25)
+end
+
+When(/^I enter my gender as male$/) do
+  your_details_page.genders.select('Male')
+end
+
+When(/^My salary is 25000$/) do
+  your_details_page.salary.set(25000)
+end
```

Comment:
There is steps for other values on age, gender and salary. Can we move this ones to the scenario outline examples? The we don't duplicated steps with the opposite value. Wdyt?